crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -49,6 +49,15 @@ impl Default for ComputedVisibility {
 #[derive(Component)]
 pub struct NoFrustumCulling;
 
+/// Collection of entities visible from the current view.
+///
+/// This component contains all app world entities which are visible from the currently
```

Comment:
```suggestion
/// This component contains all app and render world entities which are visible from the currently
```

Reply:
Actually I think it'd be less confusing to just remove "app" here. This is probably not the place to document that app and render worlds are in sync, and phrased as you did @james7132 it sounds to me like the collections contains all entities from the app world AND all entities from the render world (so, twice). Thoughts?

Reply:
```suggestion
/// This component contains all entities which are visible from the currently
```

I agree. This is actually clearer and more robust to future changes IMO.

Reply:
The Entities of both worlds are kept in sync. An Entity in one is valid in the other.